client/my-sites/sidebar/sidebar.jsx

```diff
@@ -308,14 +308,20 @@ module.exports = React.createClass( {
 			return null;
 		}
 
-		const planLink = '/plans' + this.siteSuffix(),
-			planName = ( this.isSingle() ) ? site.plan.product_name_short : '';
+		const planLink = '/plans' + this.siteSuffix();
 
 		let linkClass = 'upgrades-nudge';
+
 		if ( productsValues.isPlan( site.plan ) ) {
 			linkClass += ' is-paid-plan';
 		}
 
+		let planName = site.plan.product_name_short;
```

Comment:
I guess it's just the result of this [copy/paste](https://github.com/Automattic/wp-calypso/commit/e5b0c0c93c7cd35f44707ab84c7901f99ccd8caa#diff-3e29fa10a6f1fb8927b10336ff9ff66bL281).